tests/integration/test_trading.py

```diff
@@ -294,3 +299,795 @@ def test_estimate_position(vega_service_with_market: VegaServiceNull):
+
+    # Initialize the Market Manager
+    marketManager = ConfigurableMarketManager(
+        proposal_key_name="MM_WALLET.name",
+        termination_key_name="TERMINATE_WALLET ",
+        market_name="CRYPTO:BTCDAI/DEC22",
+        market_code="MARKET",
+        asset_name=ASSET_NAME,
+        asset_dp=0,
+        proposal_wallet_name="MM_WALLET.name",
+        termination_wallet_name="termination_wallet",
+        market_config=configWithSlippage,
+        tag="my_tag",
+        settlement_price=1000.0,
+        initial_mint=1e6
+    )
+
+    # Initialize the manager and create the market
+    marketManager.initialise(vega=vega, create_key=True, mint_key=True)
+    asset_id = vega.find_asset_id(symbol=ASSET_NAME)
+
+    for wallet in WALLETS:
+        vega.mint(
+            wallet.name,
+            asset=asset_id,
+            amount=mint_amount,
+        )
+    vega.mint(
+        PARTY_A.name,
+        asset=asset_id,
+        amount=collateral_available,
+    )
+    # Wait for the market creation to complete
+    marketManager.vega.wait_for_total_catchup()
+    market_id = vega.all_markets()[0].id
+
+    # Access the updated value
+    print(f"linear slippage factor = {configWithSlippage.linear_slippage_factor}")  
+    vega.update_network_parameter(
+            MM_WALLET.name, parameter="network.markPriceUpdateMaximumFrequency", new_value="0"
+        )
+    market_id = vega.all_markets()[0].id
+
+    vega.submit_liquidity(
+        key_name=MM_WALLET.name,
+        market_id=market_id,
+        commitment_amount=initial_commitment,
+        fee=0.002,
+        buy_specs=[("PEGGED_REFERENCE_BEST_BID", 1000, 1)],
+        sell_specs=[("PEGGED_REFERENCE_BEST_ASK",1000, 1)],
+        is_amendment=False,
+    )
+    order_id_C=vega.submit_order(
+        trading_key=PARTY_C.name,
+        market_id=market_id,
+        order_type="TYPE_LIMIT",
+        time_in_force="TIME_IN_FORCE_GTC",
+        side="SIDE_BUY",
+        order_ref="best-ask",
+        price=990,
+        volume=initial_volume,
+        wait=True,
+    )
+    vega.submit_order(
+        trading_key=PARTY_B.name,
+        market_id=market_id,
+        order_type="TYPE_LIMIT",
+        time_in_force="TIME_IN_FORCE_GTC",
+        side="SIDE_SELL",
+        price=1200,
+        volume=initial_volume,
+    )
+    vega.submit_order(
+        trading_key=PARTY_A.name,
+        market_id=market_id,
+        order_type="TYPE_LIMIT",
+        time_in_force="TIME_IN_FORCE_GTC",
+        side="SIDE_BUY",
+        price=1000,
+        volume=initial_volume,
+    )
+    #AC 0012-NP-LIPE-003: An estimate is obtained for a position with no open volume and a single limit buy order, after the order fills the mark price keeps going down in small increments and the actual liquidation takes place within the obtained estimated range. 
+    estimate_margin_open_vol_only_before_trade, estimate_liquidation_price_including_buy_orders = vega.estimate_position(
```

Comment:
We can "drop" the `estimate_margin_open_vol_only_before_trade` variable by replacing it with `_` if we don't need it.